setup.py

```diff
@@ -0,0 +1,36 @@
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+with open(os.path.join(here, 'README.md')) as f:
+    README = f.read()
+
+with open(os.path.join(here, 'CHANGELOG.rst')) as f:
+    CHANGELOG = f.read()
+
+
+REQUIREMENTS = [
+    'waitress',
+    'kinto.core',
+    'requests'
+]
+
+ENTRY_POINTS = {
+    'paste.app_factory': [
+        'main = webpush-channels:main',
```

Comment:
We should probably use ``webpush_channels`` (which should be the python module and not the python package).